src/connection-manager/latency-monitor.js

```diff
@@ -0,0 +1,242 @@
+'use strict'
```

Comment:
The latency monitor was pulled from the npm package right? Can we get attribution added, I'm not seeing it anywhere.

Reply:
looks better now ?